ui/src/overview.tsx

```diff
@@ -632,7 +666,11 @@ function ParticipantsGroup(props: {
                   "&:hover": { textDecoration: "none" },
                 }}
               >
-                {groupCountState.data?.toLocaleString()}
+                {isValid(groupCountState.data)
+                  ? groupCountState.data < 0
```

Comment:
Are you seeing counts returned that are <0?

Reply:
No, -1 was just being used as a loading status. Not actually needed here anymore though so I removed it.